Fix compilation with Qt 5.5

Because of commit 90e7cc172a7521396bb2d49720ee4ceb9a9390b3 in qtbase,
we now need to explicitly include QDataStream

Fixes #2809
https://github.com/owncloud/client/commit/ba65187ad3ec727f72da7c6b643d8e49cfcfd6ad

diff -Naur owncloudclient-1.8.1-beta1/src/3rdparty/qtsingleapplication/qtlocalpeer.cpp owncloudclient-1.8.1-beta1.tpg/src/3rdparty/qtsingleapplication/qtlocalpeer.cpp
--- owncloudclient-1.8.1-beta1/src/3rdparty/qtsingleapplication/qtlocalpeer.cpp	2015-04-17 11:31:05.000000000 +0000
+++ owncloudclient-1.8.1-beta1.tpg/src/3rdparty/qtsingleapplication/qtlocalpeer.cpp	2015-04-19 11:39:07.170140831 +0000
@@ -30,6 +30,7 @@
 #include "qtlocalpeer.h"
 
 #include <QCoreApplication>
+#include <QDataStream>
 #include <QTime>
 
 #if defined(Q_OS_WIN)
diff -Naur owncloudclient-1.8.1-beta1/src/gui/accountmigrator.h owncloudclient-1.8.1-beta1.tpg/src/gui/accountmigrator.h
--- owncloudclient-1.8.1-beta1/src/gui/accountmigrator.h	2015-04-17 11:31:05.000000000 +0000
+++ owncloudclient-1.8.1-beta1.tpg/src/gui/accountmigrator.h	2015-04-19 11:39:29.568733868 +0000
@@ -15,6 +15,7 @@
 #define ACCOUNTMIGRATOR_H
 
 #include <QStringList>
+#include <QObject>
 
 namespace OCC {
 
diff -Naur owncloudclient-1.8.1-beta1/src/libsync/cookiejar.cpp owncloudclient-1.8.1-beta1.tpg/src/libsync/cookiejar.cpp
--- owncloudclient-1.8.1-beta1/src/libsync/cookiejar.cpp	2015-04-17 11:31:05.000000000 +0000
+++ owncloudclient-1.8.1-beta1.tpg/src/libsync/cookiejar.cpp	2015-04-19 11:39:07.170140831 +0000
@@ -19,6 +19,7 @@
 #include <QFile>
 #include <QDateTime>
 #include <QNetworkCookie>
+#include <QDataStream>
 
 namespace OCC {
 
diff -Naur owncloudclient-1.8.1-beta1/src/libsync/theme.h owncloudclient-1.8.1-beta1.tpg/src/libsync/theme.h
--- owncloudclient-1.8.1-beta1/src/libsync/theme.h	2015-04-17 11:31:05.000000000 +0000
+++ owncloudclient-1.8.1-beta1.tpg/src/libsync/theme.h	2015-04-19 11:39:07.170140831 +0000
@@ -16,7 +16,7 @@
 #define _THEME_H
 
 #include "syncresult.h"
-
+#include <QObject>
 
 class QIcon;
 class QString;
